tests/python/unittest/test_gluon.py

```diff
@@ -21,7 +21,7 @@
 import mxnet as mx
 from mxnet import gluon
 from mxnet.gluon import nn
-from mxnet.test_utils import assert_almost_equal
+from mxnet.test_utils import assert_almost_equal, same
```

Comment:
why need?

Reply:
same? I'm using it in the test.

Reply:
To check that ndarray are equal. Is that not correct?

Reply:
But there is no change in the test in this file.